.chloggen/singleton-flags-and-attributes-exporter.yaml

```diff
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

Comment:
I don't think this is a breaking change since the interface is sealed, right?

Reply:
> Open to other opinions but I think the error message does not warrant a breaking change. If it were in a stable package, maybe it would warrant it, but since it's not, I'll remove the breaking notice

This is covered in the  [VERSIONING.md document](https://github.com/open-telemetry/opentelemetry-collector/blob/main/VERSIONING.md#general-go-api-considerations) (emphasis mine):

> Unless otherwise specified in the documentation, the following may change in any way between minor versions:
> - String representation. **The** String or **Error method** of any struct is intended to be human-readable and **may change its output in any way**.